src/components/VerticalNav/VerticalNavSecondaryItem.js

```diff
@@ -14,6 +14,8 @@ import VerticalNavItemHelper from './VerticalNavItemHelper';
  */
 const BaseVerticalNavSecondaryItem = props => {
   if (wrongDepth(props, 'secondary')) return correctDepth(props);
+  if (props.divider) return <VerticalNavItemHelper isDivider {...props} />;
```

Comment:
if you named the prop `isDivider` in both components, this line can simply be deleted. the next line passes all props down.